Dockerfile-periodic

```diff
@@ -11,13 +11,6 @@ RUN apt-get update -yqq \
     locales \
   && apt-get -q clean
 
-# Build locales to avoid encoding issues with Scrapy encoding
-RUN locale-gen en_GB.UTF-8
-
-ENV LC_ALL=en_GB.UTF-8
-ENV LANG=en_GB.UTF-8
-ENV LANGUAGE=en_GB.UTF-8
-
```

Comment:
why are these not needed anymore?

Reply:
We switched to a Python based docker base container which already uses utf8

Reply:
👍